### PR TITLE
eg012EmbeddedConsole implemented

### PR DESCRIPTION
**Pull request by NicklausBrain**

### CODE REVIEW

**Comment by NicklausBrain** [comment]
@serhio17 I must confess that even though API successfully returned the console URL, it opens the empty page

`The console URL is https://demo.docusign.net/Member/StartInSession.aspx?StartConsole=1&t=cb9fb0c5-2b93-47bc-8b79-c6d789cd2eac`

Please test the sample yourself as well, it is interesting what will you get.

**Comment by serhio17** [comment]
The URL https://demo.docusign.net/Member/StartInSession.aspx?StartConsole=1&t=cb9fb0c5-2b93-47bc-8b79-c6d789cd2eac redirects me to the login page where I have to login to my docusign account...

File: examples/eg012EmbeddedConsole.ps1 [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    Write-Output "$([int][ViewType]::FrontPage) - Front page"
+    Write-Output "$([int][ViewType]::EnvelopeView) - Envelope view"
+    [int]$selectedView = Read-Host "Please make a selection"
+} while (-not [ViewType]::IsDefined([ViewType], $selectedView));
+
+Write-Output ""
+Write-Output "Requesting the console view url"
+Write-Output ""
+
+$response = New-TemporaryFile
```
Comment (serhio17) [inline examples/eg012EmbeddedConsole.ps1:48]: The "$response" variable is not used in the script. Need to remove it

**Review by serhio17 — CHANGES_REQUESTED** [review]
The rest looks good

**Pull request merged**